packages/ebayui-core-react/src/ebay-file-preview-card-group/file-preview-card-group.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, FC, ComponentProps } from 'react'
+import { filterByType } from '../common/component-utils'
+import {
+    EbayFilePreviewCard,
+    EbayFilePreviewCardProps
+} from '../ebay-file-preview-card'
+
+export type EbayFilePreviewCardGroupProps = ComponentProps<'div'> & {
+    a11ySeeMoreText?: EbayFilePreviewCardProps['a11ySeeMoreText']
```

Comment:
I guess my previous comment was confusing sorry, but we still need to have `onCancel`, `onDelete` and `onMenuAction` here in the props. Those are different than the `EbayFilePreviewCard` ones since we need to pass the `index` also here